internal/services/url_normalization_settings/resource_test.go

```diff
@@ -191,7 +192,7 @@ func TestAccCloudflareURLNormalizationSettings_InvalidValues(t *testing.T) {
 			},
 			{
 				Config:      testAccCheckCloudflareURLNormalizationSettingsConfig(zoneID, "cloudflare", "invalid_scope", rnd),
-				ExpectError: regexp.MustCompile(`value must be one of: \["none" "incoming" "both"\]`),
+				ExpectError: regexp.MustCompile(`value must be one of: \["incoming" "both" "none"\]`),
```

Comment:
`ExpectError: regexp.MustCompile(`value must be one of: \["both"|"incoming"|"none"\]`),`